jenkins-agents/ansible/roles/kogito/tasks/main.yml

```diff
@@ -28,9 +28,6 @@
 - name: Install Yarn
   shell: npm install -g yarn --registry=${NPM_REGISTRY_URL}
 
-- name: Install build-chain-action
-  shell: npm install -g @kie/build-chain-action@{{buildchain_version}} --registry=${NPM_REGISTRY_URL}
-
 - name: Install build-chain-action
```

Comment:
isn't this wrong name?

Reply:
+1 - not sure if it is related to build-chain-action (and removed as well) or just a wrong name for unrelated task

Reply:
replaced by `Install npm-cli-login npm library` ! thanks